build/shade/_k-standard-goals.shade

```diff
@@ -25,9 +25,9 @@ default SAMPLES_PROJECT_GLOB = "samples/*/project.json"
   {
     E("DOTNET_BUILD_VERSION", BuildNumber);
   }
-  if (string.IsNullOrEmpty(E("DOTNET_AUTHOR")))
+  if (string.IsNullOrEmpty(E("NUGET_AUTHOR")))
   {
-    E("DOTNET_AUTHOR", AUTHORS);
+    E("NUGET_AUTHOR", 'Microsoft Open Technologies, Inc.');
```

Comment:
This is what we have today - https://github.com/aspnet/KoreBuild/blob/dev/build/shade/makefile.shade#L4